R/plot_fns.R

```diff
@@ -2512,94 +2603,94 @@ plot.totalCountFilt <- function (filter_object, min_count = NULL,
 #'
 #' @export
 #'
-plot.RNAFilt <- function (filter_object, plot_type = "library",
-                          size_library = NULL, min_nonzero = NULL,
-                          interactive = FALSE,
-                          x_lab = NULL, y_lab = NULL,
-                          x_lab_size = 11, y_lab_size = 11,
-                          x_lab_angle = 90, title_lab = NULL,
-                          title_lab_size = 14, legend_lab = "",
-                          legend_position = "right", text_size = 3,
-                          bar_width = 0.8, bw_theme = TRUE,
-                          palette = NULL) {
-
-  # @param log_total_counts logical value. Indicates if the X-axis should be on
-  #   the log scale. The default is TRUE.
+plot.RNAFilt <- function(x, plot_type = "library",
+                         size_library = NULL, min_nonzero = NULL,
+                         interactive = FALSE,
+                         x_lab = NULL, y_lab = NULL,
+                         x_lab_size = 11, y_lab_size = 11,
+                         x_lab_angle = 90, title_lab = NULL,
+                         title_lab_size = 14, legend_lab = "",
+                         legend_position = "right", text_size = 3,
+                         bar_width = 0.8, bw_theme = TRUE,
+                         palette = NULL, ...) {
+  filter_object <- x
 
   # Preliminaries --------------------------------------------------------------
 
+  # Make sure we only have valid arguments
```

Comment:
I'd argue that the argument is misleading in this case or at the very least for consistency this same logic ought to be applied to all other incidents of the ... that don't get called in the functions